heartbeat/azure-lb

```diff
@@ -141,13 +141,13 @@ lb_stop() {
                 stop_timeout=10
         fi
 
-	if lb_monitor; then
+	if lb_monitor >/dev/null 2>&1; then
```

Comment:
Seems like we should redirect the output in lb_monitor() instead.

Reply:
I think it's better to keep the stdout/stderr from lb_monitor() when it's a monitor or start operation, and only discard it in the stop operation.

We could check the operation type from within the monitor operation and send the output to /dev/null if it's a stop operation. That gives us another nested "if" though.

Reply:
It's cleaner to do it in the function, than redirect in multiple calls.

Just do a `[  ] && ` or or-version, and do it on debug-level, so we dont get "spammed" when we dont need the info.

Reply:
There's no spam really. If we get output or error during a monitor operation, then that's probably something we should know about. Under normal circumstances, we shouldn't get any stdout or stderr during the monitor or start.

The only time it regularly occurs when it shouldn't, is during the stop operation. Even during a normal, clean stop, there will be stderr from lb_monitor() (due to `kill -s 0` on a no-longer-existing PID) at some point.

I'll get an update pushed shortly.

Reply:
Wouldn't opening a fd on /dev/null and setting it as stderr & stdout at the beginning of stop be the simplest?

Reply:
It's better to use ocf_log, and set level info (for start-action) and debug (for other actions).

Reply:
We don't want to redirect everything to `/dev/null` in the lb_stop() -- only the output/error of the lb_monitor() calls.